pkg/util/progressdots/progressdots.go

```diff
@@ -45,15 +54,20 @@ func (s *ProgressDots) Start() {
 				return
 			default:
 				fmt.Print(".")
+				s.dotCounter++
 				time.Sleep(s.interval)
+				if s.dotCounter%10 == 0 {
+					s.interval = s.interval + 1*time.Second
```

Comment:
This means after 10 seconds, we are doubling the frequency of dots. so even after 30 seconds we have 2 dots in one seconds.